### ISSUE TITLE
Failing production build with fullTemplateTypeCheck & strictNullChecks

### ISSUE THREAD

**Issue by Enngage** [selvera/npm-datepicker#13]
Hi, another one :-) 

Production build with with `ts.config` option:

`  "strictNullChecks": true `

and Angular specific:

```
 "angularCompilerOptions": {
    "fullTemplateTypeCheck": true,
  },

```
Fails with: `ERROR in node_modules\@coachcare\datepicker\coachcare-datepicker.d.ts.MatYearsView.html(3,3): : Object is possibly 'null'.`

![b2](https://user-images.githubusercontent.com/7137351/42212539-b156554c-7eb6-11e8-9df8-df2bcccf7b55.png)

**Comment by Enngage** [comment]
Any chance you can fix this?

**Comment by matheo** [comment]
@Enngage I will look into this as soon as I can, been busy in my job ;)

**Comment by matheo** [comment]
@Enngage please try `0.9.6` and report back if there's a behavior change with the strict null changes ;)

**Comment by Enngage** [comment]
Awesome, thanks :) I just went on Vacation so I won't be able to check
right now, but will as soon as I can.

**Comment by matheo** [comment]
@Enngage this should be completely fixed on `@coachcare/datepicker 0.9.9`
